File: frontend/ReactMusicCommerce/src/models/AdminOrderDetailDTO.ts
export interface AdminOrderDetailDTO {
  orderDetailId: number;
  audioId: number;
  trackTitle: string;
  artistName: string;
  coverImage: string | null;
  licenseType: string;
  price: number;
  duration: number | null;
  watermarkId: string | null;
  expiredAt: string | null;
  licenseStatus: string;
}

File: backend/music/src/main/java/com/springboot/music/dto/AdminOrderDetailDTO.java
package com.springboot.music.dto;

import lombok.*;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class AdminOrderDetailDTO {

    private Integer orderDetailId;
    private Integer audioId;
    private String trackTitle;
    private String artistName;
    private String coverImage;
    private String licenseType;
    private Double price;
    private Integer duration;
    private String watermarkId;
    private LocalDateTime expiredAt;
    private String licenseStatus;
}

